templ_attr.cif

```diff
@@ -609,21 +614,90 @@ save_aniso_BIJ_su
      save_
 
 
+save_aniso_betaIJ
+
+    _definition.update           2023-02-16
+    _description.text
+;
+     These are the standard unitless anisotropic atomic displacement components
+     which appear in the structure factor term:
+
+         T = exp{-1 sum~i~ [ sum~j~ (\b^ij^ h~i~ h~j~) ] }
+
+     h = the Miller indices
+
+     The unique elements of the real symmetric matrix are entered by row.
+
+     The IUCr Commission on Nomenclature recommends against the use
+     of \b for reporting atomic displacement parameters; U is preferred [1].
+
+     Note that U^ij^ = \b^ij^/(2 pi^2^ a*~i~ a*~j~) = B^ij^/(8 pi^2^) [1].
+
+     [1] Trueblood, K. N. et al. (1996). Acta Crystallogr. A52(5), 770-781.
+;
+    _type.purpose                Measurand
+    _type.source                 Derived
+    _type.container              Single
+    _type.contents               Real
+    _units.code                  none
+     save_
+
+
+save_aniso_betaIJ2
+
+    _definition.update          2023-02-15
+    _description.text
+;
+     The [I][J] tdf elements define the overall anisotropic
```

Comment:
I know that this is copied from `save_aniso_BIJ2`, but what does 'tdf' stand for?